### ISSUE TITLE
python version

### ISSUE THREAD

**Issue by Neronjust2017** [cchen-cc/Robust-Mseg#1]
Hi, which python version did you use?

**Comment by Neronjust2017** [comment]
Hi, thanks a lot for sharing your work and the code is very clear. However, I want to know how to get these files in "load_data"? Following the instruction, I run preprocessing.py, but don't get these files. 

'''
def load_data(data_list_pth, data_pth, modality_list, batch_size, gt_flag, crop_size = 80, num_cls=5):

    with open(data_list_pth, 'r') as fp:
        rows = fp.readlines()
    pid_list = [row[:-1] for row in rows]

    cnt = 0
    for modality in modality_list:
        cnt += 1
        data_list = [data_pth+'/'+pid+'/'+modality+'_subtrMeanDivStd.tfrecords' for pid in pid_list]
        if cnt == 1:
            data_vol = tf.expand_dims(_decode_samples(data_list), axis=3)
        else:
            data_vol = tf.concat((data_vol, tf.expand_dims(_decode_samples(data_list), axis=3)), axis=3)

    data_list = [data_pth + '/' + pid + '/' + 'brainmask.tfrecords' for pid in pid_list]
    brainmask = _decode_samples(data_list)

    if gt_flag:
        data_list = [data_pth + '/' + pid + '/' + 'OTMultiClass.tfrecords' for pid in pid_list]
        label = _decode_samples(data_list)
        combine_all = tf.stack((brainmask, label), axis=3)
        combine_all = tf.concat((data_vol, combine_all), axis=3)
        combine_all = tf.random_crop(combine_all, [crop_size, crop_size, crop_size, len(modality_list)+2])
        data_vol = combine_all[:, :, :, 0]
        for m in range(1, len(modality_list)):
            data_vol = tf.concat((data_vol, combine_all[:, :, :, m]), axis=2)
        brainmask = combine_all[:, :, :, len(modality_list)]
        label = combine_all[:, :, :, len(modality_list)+1]

        data_vol = tf.expand_dims(data_vol, axis=3)
        brainmask = tf.expand_dims(brainmask, axis=3)
        label = tf.one_hot(tf.cast(label, tf.int64), depth = num_cls, axis=-1)

        data_vol_batch, brainmask_batch, label_batch = tf.train.shuffle_batch([data_vol, brainmask, label], batch_size, 50, 10, num_threads=1)
        return data_vol_batch, brainmask_batch, label_batch

    else:
        combine_all = tf.concat((data_vol, tf.expand_dims(brainmask, axis=3)), axis=3)
        combine_all = tf.random_crop(combine_all, [crop_size, crop_size, crop_size, len(modality_list)+1])
        data_vol = combine_all[:, :, :, 0]
        for m in range(1, len(modality_list)):
            data_vol = tf.concat((data_vol, combine_all[:, :, :, m]), axis=2)
        brainmask = combine_all[:, :, :, len(modality_list)]

        data_vol = tf.expand_dims(data_vol, axis=3)
        brainmask = tf.expand_dims(brainmask, axis=3)

        data_vol_batch, brainmask_batch = tf.train.shuffle_batch([data_vol, brainmask], batch_size, 50, 5, num_threads=5)
        return data_vol_batch, brainmask_batch
'''

**Comment by cchen-cc** [comment]
I used python 2.7. 
For preprocessing, each brain volume (.nii) was normalized to zero mean and uni variance within the brain tissue region. After that, the preprocessed .nii were converted to tfrecrods.

**Comment by HongTingxuan** [comment]
Hi, can you share the code to generate these files? '_subtrMeanDivStd.tfrecords','brainmask.tfrecords','OTMultiClass.tfrecords'

**Comment by Neronjust2017** [comment]
> Hi, can you share the code to generate these files? '_subtrMeanDivStd.tfrecords','brainmask.tfrecords','OTMultiClass.tfrecords'

附议，that's what I want.

**Comment by Neronjust2017** [comment]
@Halleyawoo Sorry, not yet.

**Comment by cchen-cc** [comment]
I have updated the script preprocessing.py by including a function DeepMedicFormat_crop, which was used to generate the files _subtrMeanDivStd, brainmask and OTMultiClass

**Comment by billhhh** [comment]
@Halleyawoo Hello do you have these files, can you share them with me? Really appreciate
